tests/src/availability/concourse_test.go

```diff
@@ -0,0 +1,122 @@
+package availability_test
```

Comment:
I was a bit confused by this file at first because I:
- half expected a `_test.go` file to contain another suite of tests rather than helpers (based on our other tests, rather than knowledge of Go)
- wasn't sure which of the methods were local to this file or use by the other test file

So I think it'd be clearer to make this a separate package and export the one function (I think it's only one?) as `DeploymentHasFinishedInConcourse`